website/payments/models.py

```diff
@@ -273,6 +277,12 @@ def clean(self):
     def name(self):
         return f"{self.initials} {self.last_name}"
 
+    @property
+    def can_be_revoked(self):
+        return not self.owner.paid_payment_set.filter(
```

Comment:
Can we have a test for this property in the model tests?

Reply:
Yes we can!